crates/cli/src/subcommands/dns.rs

```diff
@@ -30,49 +31,34 @@ fn get_subcommands() -> Vec<Command> {
                     .required(true)
                     .help("The top level domain that you would like to register"),
             )
-            .arg(Arg::new("identity").long("identity").short('i').help(
-                "The identity that should own this tld. If no identity is specified, then the default identity is used",
-            ))
             .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The nickname, host name or URL of the server on which to register the domain"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.